PlancakeEmailParser.php

```diff
@@ -412,16 +412,14 @@ public function getBody($returnType = self::PLAINTEXT)
         }
 
         // there could be more than one boundary
-        preg_match_all('!boundary=(.*?)[;$]!mi', $this->emailRawContent, $matches);
+        preg_match_all('/boundary=(?:|")([a-zA-Z0-9\(\)_\/+-]+)(?:|")(?:$|;)/mi', $this->emailRawContent, $matches);
```

Comment:
Could you please outline exactly what was wrong with the previous regex? This definitely looks more comprehensive than the previous one and more correct, but I'd appreciate it if you could outline some cases where the old regex didn't work but the new one does.

Also, can you please update the comment on line 414 to include the detail in the comment that was on line 419 but has been removed by your changes.